ush/set_predef_grid_params.sh

```diff
@@ -1169,7 +1169,7 @@ case ${PREDEF_GRID_NAME} in
 
   ESGgrid_WIDE_HALO_WIDTH="6"
 
-  DT_ATMOS="${DT_ATMOS:-36}"
+  DT_ATMOS="${DT_ATMOS:-60}"
 
```

Comment:
@hu5970 Ming, I see that 3-km NA RRFS uses 36s time step. CONUS RRFS has dt_atmos=40. I guess there were crashes with 40s in NA domain? Hopefully, 60s with dt_inner=30 that I tested in CONUS RRFS will not cause any crashes in NA RRFS.